src/components/blocks/InformationBlock/InformationBlock.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { render, screen } from '@testing-library/react';
+import InformationBlock from './InformationBlock';
+import type { InformationBlockProps } from './InformationBlock';
+
+describe('InformationBlock', () => {
+  describe("with a 'main' variant", () => {
+    it('renders with given text', () => {
+      const args: InformationBlockProps = {
+        variant: 'main',
+        data: [
```

Comment:
Please move this object to a variable and reuse it across all test cases. Rewrite it with spread operator if needed in particular test case